requirements.txt

```diff
@@ -1,103 +0,0 @@
-aiofiles==0.6.0
```

Comment:
Mmm?

Reply:
For some reason the file is truncated and strange things happen to it. I updated again hoping it would be resolved